tutorials/finetune_lora.md

```diff
@@ -34,7 +34,9 @@ The finetuning requires at least one GPU with ~24 GB memory (RTX 3090).
 
 This script will save checkpoints periodically to the folder `out/`.
 
-
+**Note**: LoRA can be applied to not only `query`, `key` or `value` matrices, but also to `projection`, `mlp` and classification `head`.</br>
+Accordingly to [QLoRA](https://arxiv.org/abs/2305.14314) paper (section 4): "LoRA on all linear transformer block layers are required to match full finetuning performance".</br>
+By default LoRA applied only to `query` and `value` matrices. In order to apply LoRA to other weight matrices - change config in `finetune/lora.py` accordingly.
```

Comment:
```suggestion
By default LoRA is applied only to the `query` and `value` matrices. In order to apply LoRA to other weight matrices - change the variables in `finetune/lora.py` accordingly.
```

Reply:
![giphy](https://github.com/Lightning-AI/lit-gpt/assets/58434077/315d0664-42b9-4e7d-8d24-b569420c9295)


Well, it comes to no surprise that recruiters don't particularly believe me when I say that my english is decent.

Reply:
Who needs proper English when you have proper code :wink:

Reply:
That's smart. 

<img src="https://github.com/Lightning-AI/lit-gpt/assets/58434077/5785e372-4161-4747-a29e-0ed08d7bdbd3" alt="drawing" width="200"/>